packages/cactus-plugin-ledger-connector-daml/src/main/typescript/plugin-ledger-connector-daml.ts

```diff
@@ -0,0 +1,301 @@
+import type { Express } from "express";
+import OAS from "../json/openapi.json";
+
+import {
+  GetPartiesInvolvedEndpointRequest,
+  QueryIOUEndpointRequest,
+  QueryRawIOUEndpointRequest,
+  CreateIOUEndpointRequest,
+  CreateIOUEndpointResponse,
+  ExerciseIOUEndpointRequest,
+  ExerciseIOUEndpointResponse,
+} from "./generated/openapi/typescript-axios";
+
+import {
+  IPluginLedgerConnector,
+  IWebServiceEndpoint,
+  IPluginWebService,
+  ICactusPluginOptions,
+  ConsensusAlgorithmFamily,
+  ICactusPlugin,
+} from "@hyperledger/cactus-core-api";
+import { consensusHasTransactionFinality } from "@hyperledger/cactus-core";
+import {
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+} from "@hyperledger/cactus-common";
+
+import {
+  ICreateIOUEndpointOptions,
+  CreateIOUEndpoint,
+} from "./web-services/create-iou-endpoint";
+
+import {
+  IExerciseIOUEndpointOptions,
+  ExerciseIOUEndpoint,
+} from "./web-services/exercise-iou-endpoint";
+
+import {
+  IQueryIOUEndpointOptions,
+  QueryIOUEndpoint,
+} from "./web-services/query-iou-endpoint";
+
+import {
+  IQueryRawIOUEndpointOptions,
+  QueryRawIOUEndpoint,
+} from "./web-services/query-raw-iou-endpoint";
+
+import {
+  IGetPartiesInvolvedEndpointOptions,
+  GetPartiesInvolvedEndpoint,
+} from "./web-services/get-parties-involved-endpoint";
+
+import fetch from "node-fetch";
+
+export interface IPluginLedgerConnectorDAMLOptions
+  extends ICactusPluginOptions {
+  logLevel?: LogLevelDesc;
+  apiUrl: string;
+  damlApiUrl?: string;
+}
+
+export class PluginLedgerConnectorDAML
+  implements
+    IPluginLedgerConnector<
+      CreateIOUEndpointResponse,
+      CreateIOUEndpointRequest,
+      ExerciseIOUEndpointRequest,
+      ExerciseIOUEndpointResponse
+    >,
+    ICactusPlugin,
+    IPluginWebService
+{
+  private readonly instanceId: string;
+  private readonly log: Logger;
+  private endpoints: IWebServiceEndpoint[] | undefined;
+  public readonly damlApiUrl: string;
+
+  constructor(public readonly options: IPluginLedgerConnectorDAMLOptions) {
+    this.damlApiUrl = options.apiUrl || "https://127.0.0.1:8888";
+    const level = options.logLevel || "INFO";
+    const label = "plugin-ledger-connector-daml";
+    this.log = LoggerProvider.getOrCreate({ level, label });
+
+    this.instanceId = this.options.instanceId;
+  }
+
+  public getPackageName(): string {
+    return "@hyperledger/cactus-plugin-ledger-connector-daml";
+  }
+
+  public async transact(): Promise<any> {
+    return null as any;
```

Comment:
@raynatopedrajeta Instead of returning null please throw the appropriate HTTP error from the http errors library that we use 
My recommendation for the code is this:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501